src/transaction.ts

```diff
@@ -19,9 +19,10 @@ export class Transaction {
   private readonly _afterCommitHooks: Set<AfterTransactionCommitCallback> = new Set();
   private readonly savepoints: Transaction[] = [];
   private readonly options: PartlyRequired<TransactionOptions, 'type' | 'isolationLevel' | 'readOnly'>;
-  private readonly parent: Transaction | null;
+  readonly parent: Transaction | null;
   readonly id: string;
-  private readonly name: string;
+  // TODO: this should become private, the savepoint queries should request the save point ID instead of a Transaction object
```

Comment:
I wasn't sure, but is the savepoint ID just the Transaction id?

Reply:
This one?
https://github.com/sequelize/sequelize/blob/eb0ecbe44afb2284a18595360d653f2b68c3cc5e/src/transaction.ts#L58

Reply:
Yep